src/lib/containers/row_renderers/utils/Icon.tsx

```diff
@@ -9,7 +9,9 @@ import {
   TOOL,
   STUDY,
   EXPERIMENTAL,
+  EXPERIMENTAL_TOOL,
```

Comment:
EXPERIMENTAL_TOOL is synonymous with EXPERIMENTAL (design showed "EXPERIMENTAL TOOL" as the type)